specification/portal/resource-manager/readme.md

```diff
@@ -24,18 +24,19 @@ For other options on installation see [Installing AutoRest](https://aka.ms/autor
 
 These are the global settings for the portal.
 
-```yaml
+``` yaml
 openapi-type: arm
-tag: package-2015-11-01-preview
+tag: package-preview-2015-08
 ```
 
-### Tag: package-2015-11-01-preview
 
-These settings apply only when `--tag=package-2015-11-01-preview` is specified on the command line.
+### Tag: package-preview-2015-08
 
-```yaml $(tag) == 'package-2015-11-01-preview'
+These settings apply only when `--tag=package-preview-2015-08` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2015-08'
 input-file:
-  - Microsoft.Portal/preview/2015-11-01-preview/portal.json
+  - Microsoft.Portal/preview/2015-08-01-preview/portal.json
```

Comment:
You'll also need to rename the directory that these files are in - they're still in 2015-11-01-preview.

Reply:
The files got renamed. Hover over the file name in the file section.

Reply:
Thanks for pointing that out, GitHub UI cut off the rename.